autokeras/utils.py

```diff
@@ -162,3 +166,41 @@ def read_csv_file(csv_file_path):
 def read_image(img_path):
     img = imageio.imread(uri=img_path)
     return img
+
+
+def compute_image_resize_params(data):
+    """Compute median height and width of all images in data. These values are used to resize the images at later point.
```

Comment:
Please add line breaks to the comments here.
Add "Args:" to describe data's shape.
Add "Returns:" to describe the return value. Why should we return "None, None" here?